internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -108,6 +108,25 @@ func resourceSiteRecoveryReplicatedVM() *pluginsdk.Resource {
 				Optional:         true,
 				ValidateFunc:     azure.ValidateResourceID,
 				DiffSuppressFunc: suppress.CaseDifference,
+				ConflictsWith: []string{
+					"target_availability_zone",
+				},
+			},
+			"target_availability_zone": {
+				Type:             pluginsdk.TypeString,
+				Optional:         true,
+				// Default:  "None",
+				Computed: true,
+				ForceNew: true,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
I think we can remove `DiffSuppressFunc: suppress.CaseDifference,` since we only allow number strings

Reply:
I agree, removed. thanks.